src/ui/components/SuggestedResults/index.js

```diff
@@ -0,0 +1,54 @@
+/* @flow */
+/* eslint react/no-array-index-key: 0 */
+import React from 'react';
+import escapeStringRegexp from 'escape-string-regexp';
+
+import './style.scss';
+
+type Props = {
+  query: string,
+  suggestions: Array<string>,
+  onSelect: Function,
+};
+
+const renderSuggestion = (query: string, suggestion: string, onSelect: Function) => {
```

Comment:
I'm gonna guess this is an unpopular opinion but I don't like stateless components aesthetically nor do I think we should be using them. I get it's a bit more boilerplate to write a class-based component but it's what we do most of the time and unless the entire app is some functional-paradise (for me, a nightmare!) I don't think it's worth selectively using this way of making components. I expect to see a function not a component when I see this syntax.